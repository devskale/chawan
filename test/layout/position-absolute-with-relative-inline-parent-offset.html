<!DOCTYPE html>
<!-- don't ask -->
<p>
Note: per CSS, the absolute box's sizing is explicitly UB, apparently to
accommodate for Gecko's broken rendering.  We follow Blink.
</p>
<br>
<br>test test <span style="position:relative">test test<div style="display: inline-block">adsfasdf</div><BR>
test test test test
<div style="background: blue">
<div style="display: block; position: absolute; top: 0em; left: 0; height: 100%; background: red; width: 3em">2t 2t 2</div>
asdfasd fasd
</div>
</div>
<br>
<br>
<br>
test
</span>
