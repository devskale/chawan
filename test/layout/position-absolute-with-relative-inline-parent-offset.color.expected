Note: per CSS, the absolute box's sizing is explicitly UB, apparently to
accommodate for Gecko's broken rendering. We follow Blink.



test test [48;2;255;0;0m2t 2t [49mestadsfasdf
test test [48;2;255;0;0m2     [49mest
[48;2;0;0;255masdfasd fa[48;2;255;0;0m      [48;2;0;0;255m                                                                [49m
          [48;2;255;0;0m      [49m
          [48;2;255;0;0m      [49m
          [48;2;255;0;0m      [49m
test      [48;2;255;0;0m      [49m
